keyvault/build.gradle

```diff
@@ -1,4 +1,42 @@
-apply plugin: 'java-library'
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+apply from: '../common/versioning/version_tasks.gradle'
+
+project.ext.vstsUsername = System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME") != null ? System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME") : project.findProperty("vstsUsername")
+project.ext.vstsPassword = System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_ACCESSTOKEN") != null ? System.getenv("ENV_VSTS_MVN_ANDROIDCOMMON_ACCESSTOKEN") : project.findProperty("vstsMavenAccessToken")
```

Comment:
explode if you can't find it, pls?

Reply:
This bit of code is in multiple projects.  If we're going to do this i'd want to do it at once for all projects... probably when i upgrade to latest gradle plugin.  Are you ok with leaving this as is for now?

Reply:
> This bit of code is in multiple projects. If we're going to do this i'd want to do it at once for all projects... probably when i upgrade to latest gradle plugin. Are you ok with leaving this as is for now?

Yes.